src/runtime_src/core/edge/skd/sk_daemon.cpp

```diff
@@ -274,9 +275,24 @@ static int createSoftKernelFile(uint64_t paddr, size_t size, uint32_t cuidx)
   return 0;
 }
 
+static void stacktrace_logger(const int sig)
+{
+  syslog(LOG_ERR, "%s - got %d\n", __func__, sig);
+  if (sig == SIGCHLD)
+    return;
+  void * array[25];
+  int nSize = backtrace(array, 25);
+  char ** symbols = backtrace_symbols(array, nSize);
```

Comment:
`const char **symbols`

Reply:
error: invalid conversion from 'char**' to 'const char**' [-fpermissive]
   const char **symbols = backtrace_symbols(array, nSize);
                          ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~

I did feel that this would help, not entirely sure why the petalinux compilation complains about this. Getting back char ** got it to compile again. 

In this case -fpermissive is not helping.